wmn-data.json

```diff
@@ -6542,6 +6539,118 @@
+        "m_code" : 404,
+        "known" : ["ilkarkarakurt", "sergeymeshiy"],
+        "cat" : "social"
+      },
+      {
+        "name" : "Weblancer",
+        "uri_check" : "https://www.weblancer.net/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "\"user\":",
+        "m_string" : "\"page\":\"/404\"",
+        "m_code" : 404,
+        "known" : ["alpet", "WebArtyom"],
+        "cat" : "social"
+      },
+      {
+        "name" : "Freelancehunt Freelancer",
+        "uri_check" : "https://freelancehunt.com/en/freelancer/{account}.html",
+        "e_code" : 200,
+        "e_string" : "Username:",
+        "m_string" : "User not found.",
+        "m_code" : 404,
+        "known" : ["rhythmdev_top", "vadym1232"],
+        "cat" : "social"
+      },
+      {
+        "name" : "Freelancehunt Employer",
+        "uri_check" : "https://freelancehunt.com/en/employer/{account}.html",
+        "e_code" : 200,
+        "e_string" : "Username:",
+        "m_string" : "User not found.",
+        "m_code" : 404,
+        "known" : ["rhythmdev_top", "vadym1232"],
+        "cat" : "social"
+      },
+      {
+        "name" : "Habr",
+        "uri_check" : "https://habr.com/ru/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "tm-page tm-user",
+        "m_string" : "tm-error-message",
+        "m_code" : 404,
+        "known" : ["Bo0oM", "AlhimicMan"],
+        "cat" : "social"
+      },
+      {
+        "name" : "Habr Freelancer",
+        "uri_check" : "https://freelance.habr.com/freelancers/{account}",
+        "e_code" : 200,
+        "e_string" : "user-profile profile-blocks",
+        "m_string" : "icon_user_locked",
+        "m_code" : 404,
+        "known" : ["Bo0oM", "Akloom"],
+        "cat" : "social"
+      },
+      {
+        "name" : "Habr Employer",
+        "uri_check" : "https://freelance.habr.com/freelancers/{account}/employer",
+        "e_code" : 200,
+        "e_string" : "user-profile profile-blocks",
+        "m_string" : "icon_user_locked",
+        "m_code" : 404,
+        "known" : ["aufdk", "Danvantariy"],
+        "cat" : "social"
+      },
+      {
+        "name" : "Freelance Україна",
+        "uri_check" : "https://freelance.ua/user/{account}/",
+        "e_code" : 200,
+        "e_string" : "user-profile profile-blocks",
+        "m_string" : "error-page",
+        "m_code" : 404,
+        "known" : ["mazahim33", "oleksandrseo1"],
+        "cat" : "social"
+      },
+      {
+        "name" : "FACEIT",
+        "uri_check" : "https://www.faceit.com/api/users/v1/nicknames/{account}",
+        "uri_pretty": "https://www.faceit.com/en/players/{account}",
+        "e_code" : 200,
+        "e_string" : "OK",
+        "m_string" : "user not found",
+        "m_code" : 404,
+        "known" : ["s1mple", "w0nderful"],
+        "cat" : "gaming"
+      },
+      {
+        "name" : "Beacons",
+        "uri_check" : "https://cdn.beacons.ai/profile_pictures/beacons/{account}.png",

```

Comment:
Please change this to use a string-based detection.

Reply:
The previous solution from https://github.com/WebBreacher/WhatsMyName/pull/802/commits/d3ebe76e3762cf9b720ec29f7e3754accdc2aa76 works well, but it requires bypassing Cloudflare, whereas this method does not. But I'll fix it if you want.